src/ring_theory/fractional_ideal.lean

```diff
@@ -448,7 +448,7 @@ begin
     intro y'_eq_zero,
     have : g.to_map aJ * y = 0 := by rw [←hy', y'_eq_zero, g.to_map.map_zero],
     obtain aJ_zero | y_zero := mul_eq_zero.mp this,
-    { have : aJ = 0 := g.to_map.injective_iff.1 g.injective _ aJ_zero,
+    { have : aJ = 0 := g.to_map.injective_iff.1 g.injective_to_map _ aJ_zero,
```

Comment:
Sorry, I would prefer reverting back to `injective`, because I now realise that this was in the `localization_map` and `fraction_map` namespaces. My apologies for the confusion. (As I wrote in the other comment, I had lost track of the namespaces.)